### PR TITLE
Adding search known issue for 2.0

### PR DESCRIPTION
**Pull request by dockerymick**
[Dev issue](https://github.com/open-cluster-management/backlog/issues/11862)
[Doc issue](https://github.com/open-cluster-management/backlog/issues/11923)

### CODE REVIEW

**Review by drkho — CHANGES_REQUESTED** [review]
@jlpadilla @dockerymick This bug actually happens for all resources in "Search panel"...not just for channel, subscription, and placement rule. We need to generalize the statement.

See:
Screen Shot 2021-04-28 at 10 01 24 AM
Screen Shot 2021-04-28 at 10 01 33 AM

**Review by swopebe — APPROVED** [review]
Looks good!

File: release_notes/known_issues.adoc [hunk 1]
```diff
@@ -92,6 +92,13 @@ From the console and Visual Web Terminal, users are unable to search for values
 When you are logged in as `kubeadmin` and you click the *Log out* option in the drop-down menu, the console returns to the login screen, but a browser tab opens with a `/logout` URL.
 The page is blank and you can close the tab without impact to your console.
 
+[#show-all-link-does-not-work-search]
+=== _Show all_ link does not work from the _Search_ page
+
+//2.0.10:11862
+
+The _Show all_ link is not functional from the _Search_ page. To work around this issue, you must narrow your search by seleccting more filters.
```
Comment (dockerymick) [inline release_notes/known_issues.adoc:100]: @drkho how about this update?

**Comment by jlpadilla** [comment]
@drkho The intention of the "Show all / hide" button is to hide the tiles when you have too many relationships.  It was not intended as a way to select all the tiles.

More details here: https://github.com/open-cluster-management/backlog/issues/11862#issuecomment-828711355

**Comment by dockerymick** [comment]
@drkho based on @jlpadilla comment, I believe we don’t need to doc this known issue since it’s functioning as expected. I think it is fair to doc a helpful note possibly? Let me know your thoughts

**Comment by drkho** [comment]
🆗 , thanks.

**Comment by dockerymick** [comment]
Closing this PR based on the conversation in the dev issue: https://github.com/open-cluster-management/backlog/issues/11862

**Pull request closed without merging**